src/Products/Common/Entity/Web/LoadDocumentEntity.cs

```diff
@@ -12,25 +12,25 @@ public class LoadDocumentEntity
         /// Document absolute path.
         /// </summary>
         [JsonProperty]
-        private string guid;
+        public string guid;
 
         /// <summary>
         /// Collection of the document pages with their data.
         /// </summary>
         [JsonProperty]
-        private List<PageDescriptionEntity> pages = new List<PageDescriptionEntity>();
+        public List<PageDescriptionEntity> pages = new List<PageDescriptionEntity>();
 
         /// <summary>
         /// Document print allowed flag.
         /// </summary>
         [JsonProperty]
-        private bool printAllowed = true;
+        public bool printAllowed = true;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5958475)